app/models/organisations_user.js

```diff
@@ -13,6 +14,11 @@ export default Model.extend({
   organisation: belongsTo("organisation", {
     async: false
   }),
+  user_id: attr("string"),
+  status: attr("string"),
```

Comment:
I think this branch deserves a rebase because I already added the `status` field in that file 🤔

Reply:
got this after rebasing

Reply:
weird that is shows it as an addition 🤔